src/aot/CMakeLists.txt

```diff
@@ -34,3 +34,7 @@ if(BUILD_ASAN)
   target_compile_options(bpftrace-aotrt PUBLIC "-fsanitize=address")
```

Comment:
Does that mean we should get rid of this line because it's unneeded (if I'm reading your comments correctly).

Reply:
Not at the moment, we'd have to replace `target_compile_options` by `add_compile_options` in `src/CMakeLists.txt`. Then, we can get rid of this.

I actually plan to do so in another PR to see if it finds any more ASAN errors.